### PR TITLE
fix(scheduled-tasks): no async/await for connect

### PR DESCRIPTION
**Pull request by favna**

### CODE REVIEW

**Comment by favna** [comment]
@sapphiredev pack

File: packages/scheduled-tasks/src/lib/ScheduledTaskHandler.ts [hunk 1]
```diff
@@ -9,10 +9,7 @@ export class ScheduledTaskHandler {
 
 	public constructor(options: ScheduledTasksOptions | undefined) {
 		this.strategy = options?.strategy ?? new ScheduledTaskRedisStrategy();
-	}
-
-	public async connect() {
-		await this.strategy.connect();
+		this.strategy.connect();
```
Comment (vladfrangu) [inline packages/scheduled-tasks/src/lib/ScheduledTaskHandler.ts:12]: Doesn't this affect SQS?

Comment (favna) [reply packages/scheduled-tasks/src/lib/ScheduledTaskHandler.ts:12]: Nah SQS already connected without async

File: packages/scheduled-tasks/src/lib/strategies/ScheduledTaskRedisStrategy.ts [hunk 2]
```diff
@@ -24,9 +24,9 @@ export class ScheduledTaskRedisStrategy implements ScheduledTaskBaseStrategy {
 		this.options = options?.bull ?? {};
 	}
 
-	public async connect() {
+	public connect() {
 		this.bullClient = new Bull(this.queue, this.options);
-		await this.bullClient.process((job: Job<ScheduledTaskRedisStrategyJob>) => this.run(job?.data?.task, job?.data?.payload));
+		void this.bullClient.process((job: Job<ScheduledTaskRedisStrategyJob>) => this.run(job?.data?.task, job?.data?.payload));
```
Comment (vladfrangu) [inline packages/scheduled-tasks/src/lib/strategies/ScheduledTaskRedisStrategy.ts:29]: Should we attach a .catch just for safety?

**Review by vladfrangu — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**